resources/js/requests/components/RequestsListing.vue

```diff
@@ -193,6 +193,8 @@ export default {
             break;
           case "case_title":
             field.name = "__slot:case_title";
+            // eslint-disable-next-line no-param-reassign
```

Comment:
please remove commented code

Reply:
This line is needed since it is a linter comment to enable add/set the property truncate in the parameter column.